### PR TITLE
Project controller testing

### PR DESCRIPTION
**Pull request by Markvanderuit**
Increases line coverage of ProjectController test class to 84%. 

There is a fair number of untested methods right now, that I am not entirely sure I can test:
- goToWebsite(): which happens to open a bloody webbrowser, using a static JavaFX class.
- SaveAs(): which blocks with a save as menu, using... again a static JavaFX class.
- Load(), idem ditto.
- EmptyConfigFile(), which, in its current form, would completely empty the config file and would pose a problem to users every time the test was run. Perhaps running it is a bad idea.

### CODE REVIEW

File: src/test/java/control/ProjectControllerTest.java [hunk 1]
```diff
@@ -181,4 +210,787 @@ public void start(Stage arg0) throws Exception {
...
+            );
+
+            // try to create/save the new project again with correct settings entered
+            try {
+                WhiteboxImpl.invokeMethod(projectController, "applyEdit", mouseEvent);
+                verify(projectController, atLeastOnce()).getControllerManager();
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+
```
Comment (bart-de-jonge) [inline src/test/java/control/ProjectControllerTest.java]: whitespace

**Comment by bart-de-jonge** [comment]
1 tiny comment, other than that awesome work!

File: src/test/java/control/ProjectControllerTest.java (hunk 1 above)
Comment (Markvanderuit) [inline src/test/java/control/ProjectControllerTest.java]: Whoops, removing.

**Comment by martijnjanssen** [comment]
Looks good to me!

**Pull request merged**